src/components/header/header.css

```diff
@@ -16,3 +16,7 @@
 .material-icons{
     font-size: 24px !important;
 }
+
+.menu {
+    right: -16px !important;
```

Comment:
I mean `.wrapper` which is defined and used in our `header` component. Nothing extra in `IconMenu`
https://github.com/alepop/lisk-nano/blob/2c484cfd1565634b4f682fbf207ee2bc3ce3bb42/src/components/header/header.css#L1-L4
https://github.com/alepop/lisk-nano/blob/2c484cfd1565634b4f682fbf207ee2bc3ce3bb42/src/components/header/headerElement.js#L12-L13